src/main/java/me/asleepp/SkriptItemsAdder/elements/conditions/CondCooldown.java

```diff
@@ -0,0 +1,51 @@
+package me.asleepp.SkriptItemsAdder.elements.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
+import dev.lone.itemsadder.api.CustomStack;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.Nullable;
+
+public class CondCooldown extends Condition {
```

Comment:
this name does not seem to reflect the syntax

Reply:
whoops, forgot all about that condition